internal/conf/load.go

```diff
@@ -44,8 +43,10 @@ func LoadConfigByName(name string, c interface{}) error {
 	return LoadConfigFromPath(p, c)
 }
 
-func LoadConfigFromPath(path string, c interface{}) error {
-	b, err := ioutil.ReadFile(path)
+func LoadConfigFromPath(p string, c interface{}) error {
+	prefix := getPrefix(p)
+	fmt.Println(prefix)
```

Comment:
Remove this

Reply:
After applying fixes this test started to fail - [here](https://github.com/lf-edge/ekuiper/actions/runs/1216670656) which is pretty surprising

Reply:
It is a timeout, usually rerun would solve it.